cmd/circuit/procdkr.go

```diff
@@ -53,6 +59,131 @@ func mkproc(x *cli.Context) {
 	}
 }
 
+func doRun(x *cli.Context, c *client.Client, cmd client.Cmd, path string) {
+
+	w2, _ := parseGlob(path)
+	a2 := c.Walk(w2)
+	_runproc(a2, cmd, x.Bool("tag"))
+
+}
+
+func runproc(x *cli.Context) {
+	defer func() {
+		if r := recover(); r != nil {
+			fatalf("error, likely due to missing server or misspelled anchor: %v", r)
+		}
+	}()
+	c := dial(x)
+	args := x.Args()
+
+	if len(args) != 1 && !x.Bool("all") {
+		fatalf("runproc needs an anchor argument or use the --all flag to do the entire circuit")
+	}
+	buf, _ := ioutil.ReadAll(os.Stdin)
+	var cmd client.Cmd
+	if err := json.Unmarshal(buf, &cmd); err != nil {
+		fatalf("command json not parsing: %v", err)
+	}
+	if x.Bool("scrub") {
```

Comment:
cmd.Scrub should always be true since you are executing one-off jobs so you always want the system to remove the process element automatically when the process exits.